environments/neutron-opendaylight-l2gw-sriov.yaml

```diff
@@ -0,0 +1,36 @@
+# A Heat environment that can be used to deploy OpenDaylight with L2GW and SR-IOV enabled
+resource_registry:
+  OS::TripleO::Services::NeutronOvsAgent: OS::Heat::None
+  OS::TripleO::Services::ComputeNeutronOvsAgent: OS::Heat::None
+  OS::TripleO::Services::ComputeNeutronCorePlugin: OS::Heat::None
+  OS::TripleO::Services::OpenDaylightApi: ../puppet/services/opendaylight-api.yaml
+  OS::TripleO::Services::OpenDaylightOvs: ../puppet/services/opendaylight-ovs.yaml
+  OS::TripleO::Services::NeutronL3Agent: OS::Heat::None
+  OS::TripleO::Services::NeutronSriovAgent: ../puppet/services/neutron-sriov-agent.yaml
+
+parameter_defaults:
+  NeutronEnableForceMetadata: true
+  NeutronMechanismDrivers: ['sriovnicswitch','openvswitch','opendaylight_v2']
+  NeutronServicePlugins: "odl-router_v2,networking-l2gw"
+  NeutronNetworkType: 'vxlan'
+  NeutronTenantNetworkType: vlan
```

Comment:
I dont see NeutronTenantNetworkType defined in our tht.  I do see NeutronNetworkType, which is the tenant network type.  That should be VLAN right?  and remove line 16?